easybuild/toolchains/fft/fftw.py

```diff
@@ -53,8 +53,12 @@ def _set_fftw_variables(self):
         fftw_libs = ["fftw%s" % suffix]
         if self.options['usempi']:
             fftw_libs.insert(0, "fftw%s_mpi" % suffix)
+        fftw_libs_mt = ["fftw%s" % suffix]
+        if self.options['openmp', False]:
```

Comment:
That's a very good question.

Apparently `usempi` is *always* defined, but `openmp` is not. It's subtle, and hard to pinpoint (the toolchain mechanism is... trickey).

It's probably better to do the same for `usempi`